workflow/scripts/sbs_process/apply_ic_field.py

```diff
@@ -0,0 +1,15 @@
+from lib.shared.illumination_correction import apply_ic_field
+from skimage.io import imread, imsave
+
+# load aligned image data
+aligned_image_data = imread(snakemake.input[0])
+aligned_segmentation_image = aligned_image_data[snakemake.params.segmentation_cycle - 1]
```

Comment:
I feel like "aligned_segmentation_image" deviates from the way that you are naming things. Maybe aligned_image_data_single_cycle? Idk